### ISSUE TITLE
Annotations about DCIS1 cells

### ISSUE THREAD

**Issue by luckylucky801** [wwYinYin/Celloc#3]
Hello, the annotations of the DCIS1 sc data you provided do not include annotations about tumor cells, but only nine annotations: 'B cell', 'EndoVas', 'Epithelial1', 'Epithelial2', 'Epithelial3', 'Fibroblast', 'Myeloid', 'NK/T', 'pDC'. However, annotations about tumor cells are shown in your study. Where can I see them?

**Comment by wwYinYin** [comment]
This DCIS1 sc data comes from the paper "Spatial charting of single-cell transcriptomes in tissues". Our DCIS1 sc data corresponds to his DCIS2 sc data, including nine annotations: 'B cell', 'EndoVas', 'Epithelial1', 'Epithelial2', 'Epithelial3', 'Fibroblast', 'Myeloid', 'NK/T', 'pDC'. As shown in Supplementary Figure 9a of the paper "Spatial charting of single-cell transcriptomes in tissues", 'Epithelial3' can also be annotated as tumor cells.

**Comment by xzdujiao** [comment]
Hello, I checked the raw data on GEO, GSE181254, and I noticed that in your paper, the DCIS1 scRNA and DCIS2 scRNA are likely swapped.

**Comment by wwYinYin** [comment]
You are right. In our article, we need to use DCIS2 in GSE181254 first, and then use DCIS1. In order to ensure the order in our article, we swap DCIS1 and DCIS2.
